### DISCUSSION TITLE
Directing to external sites after clicking a button

### DISCUSSION THREAD

**Discussion by FrostedDolphin** [Budibase/budibase#8181] [Help] [answered]
I am trying to create a button that is linked to an external site like discord server or slack server, looking through the options I have for buttons, it seems I can only stay inside the app. is there a way to add this functionality other than throwing the whole link on a screen?

**Comment by melohagan** [comment, marked as answer]
Hey @minitacoslayer 

You can definitely link to outside of Budibase. It is important to include the protocol, e.g. https

Through **Navigate To** action:
![Screenshot 2022-10-10 at 15 50 26](https://user-images.githubusercontent.com/101575380/194894149-17ee05e6-fc1d-48fd-8683-33ed1882663e.png)

Through link component:
![Screenshot 2022-10-10 at 15 51 38](https://user-images.githubusercontent.com/101575380/194894424-70a6c997-364c-4624-8405-777f491caa82.png)
